File: PRACTICA#2/COMPOSICION-AGREGACION/1/PYTHON/Habitacion.py
class Habitacion:
    def __init__(self, nombre, tamaño):
        self.__nombre = nombre
        self.__tamaño = tamaño

    #A
    def get_nombre(self):
        return self.__nombre

    def set_nombre(self, nombre):
        self.__nombre = nombre

    def get_tamaño(self):
        return self.__tamaño

    def set_tamaño(self, tamaño):
        self.__tamaño = tamaño

    def mostrar_info(self):
        print(f"Nombre: {self.__nombre}, Tamaño: {self.__tamaño} m²")

File: PRACTICA#2/COMPOSICION-AGREGACION/1/JAVA/Habitacion.java
public class Habitacion {
    private String nombre;
    private float tamaño;

    public Habitacion(String nombre, float tamaño) {
        this.nombre = nombre;
        this.tamaño = tamaño;
    }

    //A
    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public float getTamaño() {
        return tamaño;
    }

    public void setTamaño(float tamaño) {
        this.tamaño = tamaño;
    }
    public void mostrar_info() {
        System.out.println("Nombre: " + nombre + ", Tamaño: " + tamaño + " m²");
    }
}